submissions/emmanuelEsuong/css/styles-mediaQuery.css

```diff
@@ -0,0 +1,264 @@
+    #form {
+        overflow: hidden;
+        margin-top: 10px;
+        float: right;
+        margin-right: 3.5cm;
+    }
+    #form input {
+        width: 650px;
+        height: 1cm;
+        padding: 10px;
+        border-radius: 10px;
+        border: solid 1px;
+    }
+    #table, #form {
+        display: inline-block;
+    }
+    #table {
+        margin-right: 2cm;
+        float: right;
+        width: 1100px;
+    }
+    #form #terms {
+        width: 20px;
+        float: left;
+        margin-left: 3cm;
+    }
+    #termsLabel {
+        color: rgb(2, 209, 2);
+        font-weight: bold;
+        margin-left: 10px;
+        padding-top: 40px;
+    }
+    button {
+        border-radius: 3px;
+        border: 0px;
+        margin-left: 150px;
+        padding: 7px;
+        padding-left: 20px;
+        padding-right: 20px;
+        background-color: rgb(2, 209, 2);
+        font-weight: bold;
+        color: white;
+    }  
+    ul {
+        list-style-type: none;
+        padding: 0px;
+        overflow: hidden;
+    }
+        /* registration info */
+    #regis {
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 45px;
+        font-weight: bold;
+        color: rgb(2, 209, 2);
+    }
+    #rg, #rgs, #gt {
+        text-align: center;
+        margin-left: 7.5cm;
+        padding-bottom: 0px;
+        padding-top: 0px;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 26.5px;
+        text-align: center;
+    }
+    #table {
+        margin-right: 2cm;
+        float: right;
+        width: 1100px;
+    }
+    #signin{
+        color:rgb(2, 209, 2);
+    }
+    #supT {
+        height: 27cm;
+    }
+   /*footer*/
+    .fotnav-left a {
+        font-size: 40px;
+    }
+    .fotnav-left {
+        margin-left: 1.5cm;
+        width: auto;
+    }
+    #newsLetter label, #submit {
+        font-size: 30px;
+    }
+    #newsLetter #newsLabel {
+        float: left;
+        font-size: 40px;
+        width: 20cm;
+        margin-right: 9cm;
+        text-align: left;
+        color: white;
+    }
+    #newsLetter #input-email {
+        height: 1.3cm;
+        width: 8cm;
+    }
+    #newsLetter #submit {
+        height: 1.7cm;
+        padding-top: 0cm;
+        padding-bottom: 5px;
+        margin-top: 8px;
+        margin-left: 4px;
+        background-color: white;
+        color: rgb(2, 209, 2);
+    }
+    #icons {
+        margin-top: 25px;
+        margin-right: 10cm;
+        margin-left: 1.5cm;
+    }
+    #icons li {
+        font-size: 30px;
+        color: white;
+    }
+    #icons a {
+        height: 1cm;
+        width: 1.3cm;
+    }
+    #newsLetter {
+        float: left;
+        margin-left: 1.5cm;
+        width: 700px;
+        margin-top: 30px;
+    }
+    #fotnav li a {
+        text-decoration: none;
+        color: white;
+    }    
+    footer {
+        height: 16cm;
+        background-color:  rgb(2, 209, 2);
+        padding-top: 1cm;
+    }
+    #copyright {
```

Comment:
You used way too much styling by `ID`. I think should use `class` more. You should change some `id`s to `class`es @ElJentle

Reply:
I kind of relate more styling with "id" for now. 
Please, I'd like to use that at the moment.